### PR TITLE
Update tutorial.md

### PR DESCRIPTION
**Pull request by mrburrito**
Fixing several issues we ran into running through this tutorial with a group:
- npm init was required before installing the aws-sdk dependency
- using --save-dev resulted in a 500 error from API Gateway because the aws-sdk was not bundled with the function
- Adding the getCustomers endpoint requires a path because / is already registered for the index.js handler
- The final curl command for getCustomers needs two -- for the long form argument

Closes: #{ISSUE_NUMBER}

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -150,7 +150,8 @@ module.exports.createCustomer = async (event) => {
 You may have noticed we include an npm module to help us talk to AWS, so lets make sure we install this required npm module as a part of our service with the following command:
 
 ```bash
-npm install --save-dev aws-sdk
+npm init
```
Comment (medikoo) [inline docs/tutorial.md:153]: I don't think we should add this command here.

`npm install...` will work fine no matter if we already have `package.json` or not in a service (and in context of this tutorial we should not assume that `package.json` was not setup)

File: docs/tutorial.md [hunk 2]
```diff
@@ -227,7 +228,7 @@ getCustomers:
   handler: getCustomers.getCustomers
   events:
     - httpApi:
-        path: /
+        path: /customers
```
Comment (medikoo) [inline docs/tutorial.md:231]: What's the reason behind. Will `/` not work as expected?

**Review by medikoo — COMMENTED** [review]
@mrburrito thanks for proposal, still I'm not sure if those changes are indeed valid. Please see my comments

File: docs/tutorial.md (hunk 1 above)
Comment (mrburrito) [reply docs/tutorial.md:153]: Also, not sure if this was a change in Lambda to not bundle the SDK, but when we added `aws-sdk` as a dev dependency, the Lambdas failed saying they could not find the module. Switching to a full dependency so Serverless bundles all of the AWS libraries with the function solved the problem.

Comment (medikoo) [reply docs/tutorial.md:153]: What you approached is a problem not related to Framework, but more to the npm and node.js.

npm install install packages to first folder in which it'll find either `node_modules` or `package.json` traversing to system root.
So most likely if `npm install ..` didnt install package for you on spot, it means you had some `package.json` in folder top level (which anyway is an anti-pattern) and package was installed there.

Adding disclaimer with such explanations to tutorial feels a bit overkill (as there are probably tens of other npm or nodejs related issues user may approach, and simply it's not a place to provide a FAQ on how to address them)

Comment (mrburrito) [reply docs/tutorial.md:153]: Oh!!! That actually makes a ton of sense. Let me check and see if that was the cause, at which point, 100% agree that there's no need for that in the tutorial.

Comment (mrburrito) [reply docs/tutorial.md:153]: Confirmed. That was the problem. Adjusting PR.

File: docs/tutorial.md (hunk 2 above)
Comment (medikoo) [reply docs/tutorial.md:230]: In such case it's definitely legitimate. Thanks for improving that

**Review by medikoo — APPROVED** [review]
Thank you 👍

**Pull request merged**